pkg/agent/secondarynetwork/init.go

```diff
@@ -121,3 +152,82 @@ func createNetworkAttachDefClient(config componentbaseconfig.ClientConnectionCon
 	}
 	return netAttachDefClient, nil
 }
+
+func movePrimaryInterfaceConfigToBridge(ovsBridgeClient ovsconfig.OVSBridgeClient, primaryInterface *net.Interface, primaryInterfaceIPs []*net.IPNet, primaryInterfaceRoutes []interface{}) error {
```

Comment:
Should be possible to share code with `ConnectUplinkToOVSBridge` and `RestoreOVSBridge`? Maybe we can add two common funcs for connection and restoration in `pkgt/agent/util`.